edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml

```diff
@@ -112,7 +114,13 @@
     <deploy-libs libs.name="WPI_Native_Libraries" libs.basedir="${wpilib.native.lib}">
       <libs.local>
         <fileset id="wpiNativeLibs.local" dir="${wpilib.native.lib}">
-          <include name="**/*.so"/>
+          <include name="libHALAthena.so"/>
+          <include name="libntcore.so"/>
+          <include name="libwpilibJavaJNI.so"/>
+          <include name="libwpiutil.so"/>
+          <include name="libopencv*.so.3.1"/>
+          <include name="libcscore.so"/>
+          <include name="libopencv_java310.so"/>
```

Comment:
Should we match the order here to the C++ one, and just add the java-specific JNI ones at the end?